sync/class.jetpack-sync-module-callables.php

```diff
@@ -122,7 +122,7 @@ public function unlock_plugin_action_link_and_callables() {
 		add_filter( 'jetpack_check_and_send_callables', '__return_true' );
 	}
 
-	public function set_plugin_action_links() {
+	public function set_plugin_action_links( $current_screen ) {
```

Comment:
Why do we need to pass the current_screen into this function?

Reply:
Cause that is what the `current_screen` hook passes to the callback
https://codex.wordpress.org/Plugin_API/Action_Reference/current_screen

Reply:
@TheCrowned I believe @enejb's point is that since we are not using `$current_screen` within the function, we can omit it.

```
function jp_example_function() {
}
add_action( 'current_screen', 'jp_example_function' );
```
is a valid function hooked into `current_screen`. `$current_screen` is available to use if our function is conditional (e.g. do something specific only on particular values of `$current_screen`).

By not having unused variables, it keeps the code cleaner and helps reducing false positive and goose chases when debugging in the future.